mobile-app/app/screens/AppNavigator/screens/Loans/screens/PaybackLoanScreen.tsx

```diff
@@ -76,14 +76,20 @@ export function PaybackLoanScreen ({
     return new BigNumber(tokens.find((t) => t.id === id)?.amount ?? 0)
   }
 
+  const loanTokenOutstandingBal = new BigNumber(loanTokenAmount.amount)
+  const getAvailableLoanAmountToPay = (): string => {
+    const availableMaxPaybackAmt = BigNumber.min(loanTokenOutstandingBal, tokenBalance)
+    return availableMaxPaybackAmt.toFixed(8)
+  }
+
   const canUseOperations = useLoanOperations(vault?.state)
   const client = useWhaleApiClient()
   const token = tokens?.find((t) => t.id === loanTokenAmount.id)
   const tokenBalance = (token != null) ? getTokenAmount(token.id) : new BigNumber(0)
+  const chooseAvailableMaxLoanPaybackAmt = BigNumber.min(loanTokenOutstandingBal, tokenBalance)
```

Comment:
Possible to not use verb for variable name? especially when this is not a method
what do you think if we use `availableLoanPaybackAmt`